cloud/monitoring.rst

```diff
@@ -80,6 +80,120 @@ Failovers happen to address hardware failure, as mentioned, and also for other r
++---------------------------------------------+------------------------------------+
+| citus.locks.granted.<mode>.<locktype>.count | See below                          |
++---------------------------------------------+------------------------------------+
+| citus.mem.available                         |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.buffered                          |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.cached                            |                                    |
++---------------------------------------------+------------------------------------+
+| citus.mem.commit_limit                      | Memory currently available to      |
+|                                             | be allocated on the system         |
++---------------------------------------------+------------------------------------+
+| citus.mem.committed_as                      | Total amount of memory estimated   |
+|                                             | to complete the workload           |
++---------------------------------------------+------------------------------------+
+| citus.mem.dirty                             | Amount of memory waiting to be     |
+|                                             | written back to the disk           |
++---------------------------------------------+------------------------------------+
+| citus.mem.free                              | Amount of physical RAM             |
+|                                             | left unused                        |
++---------------------------------------------+------------------------------------+
+| citus.mem.total                             | Total amount of physical RAM       |
++---------------------------------------------+------------------------------------+
+| citus.pgbouncer_outbound.cl_active          | Active client connections          |
++---------------------------------------------+------------------------------------+
+| citus.pgbouncer_outbound.cl_waiting         | Waiting client connections         |
++---------------------------------------------+------------------------------------+
+| citus.pgbouncer_outbound.sv_active          | Active server connections          |
++---------------------------------------------+------------------------------------+
+| citus.pgbouncer_outbound.sv_idle            | Idle server connections            |
++---------------------------------------------+------------------------------------+
+| citus.pgbouncer_outbound.sv_used            | Server connections idle more       |
+|                                             | than server_check_delay            |
++---------------------------------------------+------------------------------------+
```

Comment:
I’m not fully sure that description is correct - is that straight from the pgbouncer docs?

Reply:
I shortened the description slightly, but yes. https://pgbouncer.github.io/usage.html